traceevents/src/main/java/com/tomtom/kotlin/traceevents/Tracer.kt

```diff
@@ -333,48 +398,17 @@ public class Tracer private constructor(
                     ?.also { parameterNamesCache[method] = it }
             }
         )
-
-        /**
-         * Log the event to the standard logger here, rather than on a consumer thread, to make
-         * sure the order of events and other log messages remains logical.
-         */
-        if (loggingMode == LoggingMode.SYNC) {
-            val logLevel = simpleLogFunctionNameToLogLevel(event.eventName)
-            if (logLevel != null) {
-                if (!useSimpleLogFunction(logLevel, logTag, args)) {
-
-                    // Signal the listener an incorrect signature was found.
-                    proxy.incorrectLogSignatureFound()
-                }
-            } else {
-
-                // Only format the message for non-standard Log events. Use the annotated log level.
-                TraceLog.log(
-                    logLevelAnnotation, logTag,
-                    "event=${
-                        createLogMessage(
-                            event,
-                            includeTime = false,
-                            includeExceptionStackTrace = includeExceptionStackTraceAnnotation,
-                            includeTaggingClass = includeTaggingClassAnnotation,
-                            includeFileLocation = includeFileLocationAnnotation,
-                            includeEventInterface = includeEventInterfaceAnnotation
-                        )
-                    }"
-                )
-            }
-        }
         offerTraceEvent(event)
         return null
     }
 
     // Helpers to get the annotation values. Function level overrides interface level.
-    private fun logLevelFromAnnotation(method: Method) =
+    private fun logLevelAnnotation(method: Method) =
```

Comment:
The new name implies the function runs a `LogLevelAnnotation` and not a log level. Proper name is `Method.getLogLevel()`